src/Pages/Detail/DetailSections/Categories.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import GoodsDetail from './CategoriesSections/GoodsDetail';
+import GoodsReview from './CategoriesSections/GoodsReview';
+import './DetailSection.scss';
+
+class Categories extends React.Component {
+  detailRef = React.createRef();
+  reviewRef = React.createRef();
+
+  focusPage = e => {
+    const { name } = e.target;
+    this[name].current.scrollIntoView();
+  };
+
+  render() {
+    const { goods } = this.props;
+
+    return (
+      <div className="categories">
+        <div className="categories-title">
+          <div className="categorie-list seleted">
+            <button name="detailRef" onClick={e => this.focusPage(e)}>
```

Comment:
```suggestion
            <button name="detailRef" onClick={this.focusPage}>
```
- 동일한 표현입니다
- 매번 다른 함수 인스턴스를 생성하기 때문에 성능상 불리한 부분이 있습니다. 가급적 클래스의 메서드를 사용해주세요. 클래스의 메서드는 처음 초기화될 때 한번 생성됩니다

Reply:
네 전체적으로 확인하여 수정 처리하겠습니다.